New package: Rete v1.1.0

JuliaRegistrator opened the pull request:
- Registering package: Rete
- Repository: https://github.com/MarkNahabedian/Rete.jl
- Created by: @MarkNahabedian
- Version: v1.1.0
- Commit: a16548b758717a5d42d92dd9af77ce1b9d32cc78
- Reviewed by: @MarkNahabedian
- Reference: https://github.com/MarkNahabedian/Rete.jl/issues/5#issuecomment-2234708480
- Description: An inplementation of the Rete knowledge based system shell.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
This is the initial release of this package, which provides a platform for implementing discrimination networks based on the Rete algorithm.  See traditional Artificial Intelligence texts like [Russell and Norvig](https://aima.cs.berkeley.edu/) for a description.

 The examples in the documentation are simpleminded, but another package, https://github.com/MarkNahabedian/SquareDanceReasoning.jl, makes extensive use of it.

This package does not implement unification or any pattern matching.

Forward chaining rules join those facts that match the rule's input fact types and run Julia code to filter the input facts and output new facts.

There is no retraction mechanism.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835989d55d050dd1e335de5b8cd51cc07e7722f9b9a0131ee2babd53884de23e448b1dc7f2f7fa9032268d1590d5a2c823de892064ceda5800d24c1c2fd8ebfd76d7d069faa845b77938793c42a6fcb8b4d42fb845efb5dc595b16458d6d14bc10a93b541c6d3df6c6e99ad883f374ef26ee11bf96d5da46d7c732a210f1d48a5045e -->

goerz commented:
The name seems fine to me. It will need a manual merge, of course, so post a message in the `#pkg-registration` channel on Slack after the three-day waiting period ends.

It might be good to make the text "Rete algorithm" in the README a link to https://en.wikipedia.org/wiki/Rete_algorithm or whatever website you think provides the best explanation of the method implemented in this package.

goerz commented:
Oh, you do need to fix

> The compat entry for julia is unbounded.

before this can be merged.

MarkNahabedian commented:
Take a look now.  It should be fixed at 1.10.

On Jul 17, 2024 3:41 PM, Michael Goerz ***@***.***> wrote:

Oh, you do need to fix

The compat entry for julia is unbounded.

before this can be merged.

—
Reply to this email directly, view it on GitHub<https://github.com/JuliaRegistries/General/pull/111228#issuecomment-2234101601>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIBY3E2AJAKW25CUSQ4JCELZM3CFJAVCNFSM6AAAAABLA4LPPOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMZUGEYDCNRQGE>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

goerz commented:
You'll have to retrigger the registration for it to take effect

JuliaRegistrator committed:
New package: Rete v1.1.0

UUID: 3d46c559-9b3d-4aba-9ca1-48ff7627231e
Repo: https://github.com/MarkNahabedian/Rete.jl.git
Tree: 31aea1ffa3fe2b60fab2d4891ff3fe218f1a0aea

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
